src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/takecard/Take.kt

```diff
@@ -0,0 +1,11 @@
+package org.wycliffeassociates.otter.jvm.app.widgets.takecard
+
+import java.io.File
+import java.util.*
+
+data class Take(
```

Comment:
is there a reason this isn't in common, or using one from common?

Reply:
We needed different things that was in the one in common. I put it here for now until we nail down exactly what properties should be in the class, at which point we can update the one in common.